recipe/meta.yaml

```diff
@@ -10,10 +10,11 @@ source:
   sha256: caf47ac4c6346eef47fc11e799adbeaf645921c712cc099cf2530560e7ecde44
   patches:
     - 0001-patch-sphinx-4-support.patch
+    - 0002-patch-jinja-markup-deprecation.patch
```

Comment:
Have we tried PRing these [upstream]( https://foss.heptapod.net/doc-utils/cloud_sptheme )?

Reply:
Yes. The library appears to be abandoned.  We haven't had any activity from the author the last year. 

https://foss.heptapod.net/doc-utils/cloud_sptheme/-/issues/45#note_158949
https://foss.heptapod.net/doc-utils/cloud_sptheme/-/issues/47

Maybe I should fork the repository on GitHub, but that would really be a new library.